tools/code_format/check_format.py

```diff
@@ -696,6 +696,15 @@ def checkSourceLine(line, file_path, reportError):
         reportError("Don't call grpc_init() or grpc_shutdown() directly, instantiate " +
                     "Grpc::GoogleGrpcContext. See #8282")
 
+  if not line.startswith("using"):
+    smart_ptr_m = re.search("std::(unique_ptr|shared_ptr)<(.*?)>", line)
```

Comment:
Is this a bit too strict? I understand now wanting to have verbose unique_ptr for commonly used types, but the occasional use of unique_ptr, in particular for non-Envoy types, makes sense.

Reply:
I’m going to add an allowed type list (eg. [int, std::string]) and report an error for the types which are not included in that list. I suppose it handles normal use cases well.